osf-cas/templates/configmap.yaml

```diff
@@ -677,10 +700,42 @@ services/cas-203948234207100.json: |-
     "@class": "org.apereo.cas.services.RegexRegisteredService",
     "serviceId": "^https://{{ .Values.casDomain | replace "." "\\\\." }}/.*",
     "name": "",
-    "description": "OSF CAS is the centralized authentication service for OSF",
+    "description": "",
     "id": 203948234207100,
     "evaluationOrder": 10,
-    "logo": "/images/osf-cas-banner.png",
+    "logo": "/images/osf-banner.png",
+    "attributeReleasePolicy": {
+      "@class": "org.apereo.cas.services.ReturnAllowedAttributeReleasePolicy",
+      "allowedAttributes": [
+        "java.util.ArrayList",
+        [
+          "givenName",
+          "familyName",
+          "username"
+        ]
+      ]
+    },
+    "accessStrategy": {
+      "@class": "org.apereo.cas.services.DefaultRegisteredServiceAccessStrategy",
+      "delegatedAuthenticationPolicy": {
+        "@class": "org.apereo.cas.services.DefaultRegisteredServiceDelegatedAuthenticationPolicy",
+        "allowedProviders": [
+          "java.util.ArrayList",
+          []
+        ],
+        "permitUndefined": false
+      }
+    }
+  }
+services/cas-203948234207101.json: |-
+  {
+    "@class": "org.apereo.cas.services.RegexRegisteredService",
+    "serviceId": "^/(login|logout)\\?.*",
+    "name": "",
+    "description": "",
+    "id": 203948234207101,
+    "evaluationOrder": 10,
+    "logo": "/images/osf-banner.png",
```

Comment:
The diff is stupid. A new service definition `cas-203948234207101.json` is added.

In addition, for all existing services, update a few properties to support:
* New FE banner-only / shield-with-name design
* Two institutions using `pac4j-cas` delegation